packages/@blockprotocol/type-system-node/.gitattributes

```diff
@@ -0,0 +1 @@
+src/* linguist-generated=true
```

Comment:
Let's avoid `.gitattributes` in subfolders to make repo config more transparent and easier to manage (at least for now). One source of truth helps simplify the set of rules and see the whole picture at a glance. If any folder can configure itself, that's quite a lot of extra scanning work for developers.

Reply:
I've combined and moved to the top-level in a new file as we didn't seem to have one